tools/version.rb

```diff
@@ -209,7 +215,7 @@ def get_anitya_id(name, homepage)
                                       # If a source_url exists check if
                                       # that line has 'version' in it.
                                       elsif !@pkg.source_url.nil?
-                                        if `grep source_url #{filename} | grep '\#{version}'`.empty?
+                                        if `grep source_url #{filename} | grep '\#{version'`.empty?
```

Comment:
Missing ending `}`?  Also, should we use double quotes to extrapolate?

Reply:
EDIT: Are you looking for the literal `#{version` string?

Reply:
grep finds that stuff, but if the variable string is interpreted in ruby we do not notice that.